code/modules/projectiles/gun.dm

```diff
@@ -203,6 +205,20 @@
 	if(HULK in M.mutations)
 		to_chat(M, "<span class='danger'>Your fingers are much too large for the trigger guard!</span>")
 		return 0
+	if(shootback)
+		var/obj/P = consume_next_projectile()
+		var/targetedlimb = pick(BP_HEAD, BP_TORSO)
+		if(P)
+			if(process_projectile(P, user, user, targetedlimb))
+				to_chat(user, "<span class='warning'>UNAUTHORIZED USER DETECTED.</span>")
+				user.visible_message(
+					"<span class='danger'>\The [src] buzzes, firing by itself into [user]'s [targetedlimb]!</span>",
+					"<span class='danger'>\The [src] buzzes, and as your grip loosens on it, it fires point-blank into your [targetedlimb]!</span>"
```

Comment:
`you lose your grip as the [src] buzzes, firing straight into your [targetedlimb]!` flows a bit better.

Reply:
thank you for your input